### ISSUE TITLE
Q: Intended approach to make all stored metadata in a repository inspectable

### ISSUE THREAD

**Issue by mih** [datalad/datalad-metalad#86]
This is likely possible with the metadata model helpers. However, this will be a standard maintenance task AFAICS, so it should be accessible somehow -- even if just as a note in a docstring. Did not find it, but I am also unsure re what would be a good place to put this information without a command that does it in -metadalad.

**Comment by bpoldrack** [comment]
In my estimation we'll need some(?) helpers/command(s) for that.
However, I think this:

> but I am also unsure re what would be a good place to put this information without a command that does it in -metadalad.

is a more general issue with current datalad doc/help generation. Man pages are on thing, but since they are python packages, there needs to be something `datalad datalad-metalad --help`. Some general entry point that extensions can use for conceptual docs.

**Comment by mih** [comment]
If we will get command, there is a natural place to put such docs. However, I quite like the idea of extension doc entrypoints. Dunno of doable with argparse. Maybe it needs to become:

```
datalad help datalad-metalad
```

**Comment by bpoldrack** [comment]
> `datalad help datalad-metalad`

Yeah, may be even better as is provides the option to add more "pages" that are not directly tied to a command.
